test/ad/flows.jl

```diff
@@ -1,26 +1,26 @@
 @testset "PlanarLayer" begin
     # logpdf of a flow with a planar layer and two-dimensional inputs
-    test_ad(randn(7)) do θ
+    test_ad(randn(7), (:EnzymeForward,)) do θ
```

Comment:
Just noting the same thing as on Slack, that for me these pass locally. Not sure what's going on.